docs/learn-registrar.md

```diff
@@ -0,0 +1,143 @@
+and select your account to submit the `identity -> requestJudgement(reg_index, max_fee)`
+transaction. This will request the registrar to validate the information you set on-chain earlier.
+
+```
+reg_index - the position of the registrar. For W3F, use 0
+
+max_fee - the amount of DOT you are going to pay for the registrar. For W3F, it does not cost anything, so 0.
+```
+
+> Friendly reminder: Once you have submitted "requestJudgement" transaction, if you are not able to
+> complete the verification procedure in an hour, you would have to submit the "clearidentity"
+> transaction first and then redo the whole thing from the beginning again.
+
+## Element Verification
+
+Since I have provided the Element, Twitter, and Email information in this example, I would start to
+receive the verification requests from those platforms. As for Element, an invitation will be sent
+by the bot named "W3F Registrar Verification".
+
+![registrar](assets/registrar/5.jpg)
+
+Once you accept the invitation, you should see the following information.
+
+![registrar](assets/registrar/6.jpg)
+
+Then go to [Sign and Verify](https://polkadot.js.org/apps/#/signing) under the Developer tab in the
+PolkadotJS and select your account, paste the "Challenge" data to the "sign the following data"
+field and click "Sign message".
+
+![registrar](assets/registrar/7.jpg)
+
+Copy the result of the "signature of supplied data" and paste it to the W3F Registrar Verification
+chat.
+
+![registrar](assets/registrar/8.jpg)
+
+If the information is correct, you should see a message like the above image that indicates your
+address has been verified. This basically proves you are the owner of the account.
+
+## Email Verification
+
+Next, you should receive an email called "W3F Registrar Verification Service". Below is an example
+for reference.
+
+![registrar](assets/registrar/9.jpg)
+
+You would do what you did in the above again. Copy the "Challenge" data and go to
+[Sign and Verify](https://polkadot.js.org/apps/#/signing) under the Developer tab in the PolkadotJS
+and select your account, paste the "Challenge" data to the "sign the following data" field and click
+"Sign message"
+
+![registrar](assets/registrar/10.jpg)
+
+And reply with your signed data only in the email. Then click "Send".
+
+> Note: Do not add anything in the email except the signed data.
+
+![registrar](assets/registrar/11.jpg)
+
+Wait for 1 to 2 minutes. You would receive another email that shows your email has been verified
+successfully.
+
+## Twitter Verification
+
+Lastly, if you have provided Twitter handle, you would have to follow
+[@w3f_registrar](https://twitter.com/w3f_registrar) first.
+
+![registrar](assets/registrar/12.jpg)
+
+And then wait for 1 to 2 minutes, the Twitter bot would send a challenge message to you.
+
+![registrar](assets/registrar/13.jpg)
+
+Then you would do the same thing as the previous step to go to the
```

Comment:
Is it good practice to explain the "Sign and Verify" process multiple times?

Reply:
Thanks. just updated ;)